README.rst

```diff
@@ -118,6 +182,132 @@ To stop fiware-cloto, execute:
+
+API Reference Documentation
+---------------------------
+
+- `FIWARE Policy Manager v1 (Apiary)`__
+
+__ `FIWARE Policy Manager - Apiary`_
+
+
+Testing
+=======
+
+Unit tests
+----------
+
+To execute the unit tests, you must set the environment variable pointing to the settings_test file.
+Then you can use coverage to execute the tests and obtain the percentage of lines coveved by the tests.
+
+::
+
+    $ export DJANGO_SETTINGS_MODULE=settings.settings_tests
+    $ coverage run --source=cloto,orion_wrapper,environments manage.py test
+
+End-to-end tests
+----------------
+
+In order to check that fiware-cloto is up and running, you can execute a GET request similar to:
+
+::
+
+    curl -v -H 'X-Auth-Token: $AUTH_TOKEN' http://$HOST:8000/v1.0/$TENANT_ID/
+
+Where:
+**$AUTH_TOKEN**: is a valid token owned by the user. You can request this token from keystone.
+**$HOST**: is the url/IP of the machine where fiware facts is installed, for example: (policymanager-host.org, 127.0.0.1, etc)
+**$TENANT_ID**: is a tenantId of the user, for example: 6571e3422ad84f7d828ce2f30373b3d4
+
+the response should be similar to:
+
+::
+
+    {
+        "owner": "Telefonica I+D",
+        "doc": "https://forge.fi-ware.org/plugins/mediawiki/wiki/fiware/index.php/Policy_Manager_Open_RESTful_API_Specification",
+        "runningfrom": "15/08/14 10:12:45",
+        "version": "1.7.0",
+        "windowsize": 2
+    }
+
+Please refer to the `Installation and administration guide
+<doc/admin_guide.rst#end-to-end-testing>`_ for details.
+
+Acceptance tests
+----------------
+
+ Requirements
+
+  - Python 2.7 or newer
+  - pip installed (http://docs.python-guide.org/en/latest/starting/install/linux/)
+  - virtualenv installed (pip install virtalenv)
+  - Git installed (yum install git-core / apt-get install git)
+
+ Environment preparation:
+
+  - Create a virtual environment somewhere, e.g. in ENV (virtualenv ENV)
+  - Activate the virtual environment (source ENV/bin/activate)
+  - Change to the test/acceptance folder of the project
+  - Install the requirements for the acceptance tests in the virtual environment (pip install -r requirements.txt --allow-all-external).
+  - Configure file in fiware-cloto/tests/acceptance_tests/commons/configuration.py adding the keystone url, and a valid, user, password and tenant ID.
+
+ Tests execution
+
+  Change to the fiware-cloto/tests/acceptance_tests folder of the project if not already on it and execute:
+  ::
+
+        lettuce_tools
+
+In the following document you will find the steps to execute automated
+tests for the Policy Manager GE:
+
+- `Policy Manager acceptance tests <cloto/tests/acceptance_tests/README.md>`_
+
+
+Advanced topics
+===============
+
+- `Installation and administration <doc/admin_guide.rst>`_
+- `User and programmers guide <doc/user_guide.rst>`_
+- `Open RESTful API Specification <doc/open_spec.rst>`_
+- `Architecture Description <doc/architecture.rst>`_
 
```

Comment:
Add the section *Support* before *License* with the content:

Support
======

Ask your thorough programming questions using stackoverflow and your general questions on FIWARE Q&A. In both cases please use the tag fiware-bosun

Top